wye/templates/thankyou.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
```

Comment:
Can we plan to use the same form html with if else condition to avoid new HTML file ?